doc/reference/stdio.md

```diff
@@ -622,6 +623,15 @@ The separator is either a single character or a list of characters.
 If `include-sep?` is true, then the separator is include in the stream.
 If the separator is not encountered within `max-chars`, then an error is raised.
 
+#### BufferedReader-put-back
+```scheme
+(BufferedReader-put-back buf previous-input)
+  buf := BufferedReader
+  previous-input := u8 or list of u8
+```
+
+Puts back one or more previously read bytes.
```

Comment:
The list, in which order?
How long can it be?
What if I try to put-back stuff that was never there?
What if I read so much from the buffer that it got refilled and those characters were flushed and there is no space for all the characters you try to put back in the current buffer?
When is the call guaranteed to succeed or guaranteed to fail?

Reply:
yes it is guaraneed to succed, the buffer will grow as necessary.
The order of the bytes is the natural one, oldest byte first.
And it's ok if you are actually insert your own bytes that you didnt really read before, the buffer wont mind.

Reply:
Added notes in the interface specification.